src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/CSIPServiceTest.kt

```diff
@@ -155,13 +156,13 @@ internal class CSIPServiceTest {
   }
 
   @Nested
-  @DisplayName("POST /csip-records/{cspReportId}/referral/safer-custody-screening")
+  @DisplayName("POST /csip-records/{dpsReportId}/referral/safer-custody-screening")
   inner class CreateCSIPSCS {
     private val dpsCSIPId = "a1b2c3d4-e5f6-1234-5678-90a1b2c3d4e6"
 
     @BeforeEach
     internal fun setUp() {
-      csipApi.stubCSIPInsertSCS(dpsCSIPId = dpsCSIPId)
+      csipApi.stubCSIPSCSInsert(dpsCSIPId = dpsCSIPId)
```

Comment:
Have noticed inconsistency with some of my naming conventions -CSIP/Csip/dps - will tidy up in a separate PR